composer.json

```diff
@@ -12,21 +12,24 @@
         }
     ],
     "require": {
-        "php": "^5.6|^7.0",
+        "php": "^7.1",
         "symfony/framework-bundle": "^2.8|^3.0",
-        "sonata-project/seo-bundle": "^1.1.3|^2.0",
-        "doctrine/collections": "^1.0"
+        "sonata-project/seo-bundle": "^2.2",
+        "doctrine/collections": "^1.0",
+        "symfony/symfony": "^2.8",
+        "phpcr/phpcr-utils": "^1.2.9"
     },
     "require-dev": {
-        "symfony-cmf/testing": "^2.0",
-        "doctrine/phpcr-odm": "^1.4.2",
```

Comment:
we now install a too old version of phpcr-odm it seems

Reply:
oh second wasn't wanted